lib/stdlib/doc/src/gen_statem.xml

```diff
@@ -2472,31 +2472,41 @@ handle_event(_, _, State, Data) ->
 	  of the <c>gen_statem</c>.
 	  If a process with the specified <c><anno>ServerName</anno></c>
 	  exists already, this function returns
-	  <seetype marker="#start_ret"><c>{error,{already_started,Pid}}</c></seetype>,
-	  where <c>Pid</c> is the <c>pid()</c> of that process.
+	  <seetype marker="#start_ret"><c>{error,{already_started,OtherPid}}</c></seetype>,
+	  where <c>OtherPid</c> is the <c>pid()</c> of that process.
 	</p>
 	<p>
-	  If <c>Module:init/1</c> fails with <c>Reason</c>,
-	  this function returns
-	  <seetype marker="#start_ret"><c>{error, Reason}</c></seetype>.
+	  If
+          <seemfa marker="#Module:init/1"><c>Module:init/1</c></seemfa>
+          fails with <c>Reason</c>, this function returns
+	  <seetype marker="#start_ret"><c>{error,Reason}</c></seetype>.
 	  If <c>Module:init/1</c> returns
-	  <seetype marker="#start_ret"><c>{stop, Reason}</c></seetype>,
-	  <seetype marker="#start_ret"><c>{shutdown, Reason}</c></seetype>
-	  or
-	  <seetype marker="#start_ret"><c>ignore</c></seetype>,
-	  the process is terminated and this function
-	  returns
-	  <seetype marker="#start_ret"><c>{error,Reason}</c></seetype>
-	  or
-	  <seetype marker="#start_ret"><c>ignore</c></seetype>,
-	  respectively.
-          An exit signal with the same <c>Reason</c> (or <c>normal</c> if
-          <c>Module:init/1</c> returns <c>ignore</c>) is set to linked processes
-          and ports, including the process calling <c>start_link/3,4</c>.
-	</p>
-	<p>The difference between returning <c>{stop, Reason}</c> and
-	<c>{error, Reason}</c> (from <c>Module:init/1</c>) is that
-	<c>error</c> results in a graceful ("silent") termination. </p>
+	  <seetype marker="#init_result"><c>{stop,Reason}</c></seetype> or
+	  <seetype marker="#init_result"><c>{error,Reason}</c></seetype>
+	  the process is terminated and this function returns
+	  <seetype marker="#start_ret"><c>{error,Reason}</c></seetype>.
+	  If <c>Module:init/1</c> returns
+	  <seetype marker="#init_result"><c>ignore</c></seetype>
+	  the process is terminated and this function returns
+	  <seetype marker="#start_ret"><c>ignore</c></seetype>.
+        </p>
+        <p>
+          The process exits with <c>Reason</c> unless
```

Comment:
Could do with re-asserting that what "the" process is here. At least my reference had expired once i read this far.